.github/auto-label.yml

```diff
@@ -0,0 +1,5 @@
+product: true
+staleness:
+  pullrequest: true
+  old: 30
```

Comment:
PR will be marked as "stale: old" if older then 30 days since created and "stale: critical" if older than 60 days.
Feel free to comment.